### DISCUSSION TITLE
SetDefaultPause > SetDefaultGap

### DISCUSSION THREAD

**Discussion by chenlung** [URUWorks/TeroSubtitler#73] [General]
In en-US.xml, we have this: `<actSetDefaultPause Hint="only if the gap is too short">Set default pause</actSetDefaultPause>`

'Gap' seems the right word to use instead of 'Pause' (even the tooltip says it). Should it not be `<GapTooShort>Gap too short</GapTooShort>`?

Using Tero Subtitler alpha (1.0.0.0) - Mac 10.11.6.

**Comment by URUWorks** [comment]
Corrected v1.0.0.9.
